proxies/live/apiproxy/policies/AssignMessage.SetRefreshTokenAuthProvider.xml

```diff
@@ -1,5 +1,6 @@
 <AssignMessage name="AssignMessage.SetRefreshTokenAuthProvider">
-  <IgnoreUnresolvedVariables>false</IgnoreUnresolvedVariables>
+<!-- Ignore as unresolved variable due to invalid refresh token handled by GenerateRefreshToken -->
```

Comment:
When an invalid refresh token is provided the flow continues.
Means access token and refresh token expiry times are set to nhs login. 
Flow raises fault at GenerateRefreshToken due to invalid refresh token
This behaviour follows what is currently happening on master. If incorrect can raise separate ticket